### ISSUE TITLE
How to build a Cloud-native Application

### ISSUE THREAD

**Issue by Kikiodazie** [section-engineering-education/engineering-education#2837]
### Proposed title of article
How to build a Cloud-native Application

### Introduction paragraph (2-3 paragraphs):
Industries are moving at the fastest pace ever. Customers are demanding even more, and businesses want to innovate and keep up with that pace. The software runs the world, and in other to keep up with the demand of customers, companies need to build scalable software faster than ever. 

Cloud-native application development can help businesses build to meet customers demand. Cloud-native development is an approach to building, running and updating responsive, scalable and fault-tolerant apps on the cloud (public, private, or hybrid) quickly while improving quality and reducing risk,

### Goal
In this guide, you will learn how to apply modern development techniques and tools to build a Cloud-native application from a language-agnostic perspective using a fictional app idea. 

### Key takeaways: 

- Understand the concept of Cloud-native application development.
- Understand how the Cloud-native development model works
- Know when to use the Microservices architecture.
- See how to apply techniques like containers and use tools like Docker to build a Cloud-native application.

### References: 
N/A

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by ahmadmardeni1** [comment]
To help avoid a backlog queue we are only accepting one article topic/PR approved at a time 👍 can be reopened once the first topic is merged. @Kikiodazie

**Comment by Kikiodazie** [comment]
> To help avoid a backlog queue we are only accepting one article topic/PR approved at a time  can be reopened once the first topic is merged. @Kikiodazie

Ohh. Thanks for the update. I will do well to remind you once my previous topic is merged. Have a great day @ahmadmardeni1.

**Comment by Kikiodazie** [comment]
> > To help avoid a backlog queue we are only accepting one article topic/PR approved at a time  can be reopened once the first topic is merged. @Kikiodazie
> 
> Ohh. Thanks for the update. I will do well to remind you once my previous topic is merged. Have a great day @ahmadmardeni1.

Heyyyyyy @ahmadmardeni1. Hope you good?

I am here to remind you about this topic :pleading_face: . Please do check it out.

I hope this meets you well. Have a great week ahead. 

Regards,
Divine.

**Comment by ahmadmardeni1** [comment]
Hey, @Kikiodazie Sure thing!

**Comment by Kikiodazie** [comment]
Yaaaayy 🕺

**Comment by ahmadmardeni1** [comment]
After some careful consideration, I found out that this topic is already covered in other blogs.

Can you please inform me how you are planning to make it unique? @Kikiodazie

**Comment by Kikiodazie** [comment]
> After some careful consideration, I found out that this topic is already covered in other blogs.
> 
> Can you please inform me how you are planning to make it unique? @Kikiodazie

Hey @ahmadmardeni1, so sorry for late reply. 

Why I submitted this topic idea is that, I was part of the cloud-native foundation scholarship by Udacity and I thought it would be great if I shared the knowledge I got. 

Seeing the fact that it's been covered by many other blogs, I guess there isn't a need to approve it. 


I hope this meets you well. Cheers.

**Comment by ahmadmardeni1** [comment]
Great! @Kikiodazie 
Can you add a unique idea to your article that makes it unique and different from other blogs?

**Comment by Kikiodazie** [comment]
> Great! @Kikiodazie
> Can you add a unique idea to your article that makes it different from other blogs?

Hey @ahmadmardeni1 again so sorry for late reply. 

Yes I believe so. Currently don't have that unique idea right now would have to go through other blogs and content and see what’s missing that is vital and will make the article special. 

But yes I believe I can.

I hope all this meets you well. Again sorry for late reply.

**Comment by hectorkambow** [comment]
> > Great! @Kikiodazie
> > Can you add a unique idea to your article that makes it different from other blogs?
> 
> Hey @ahmadmardeni1 again so sorry for late reply.
> 
> Yes I believe so. Currently don't have that unique idea right now would have to go through other blogs and content and see what’s missing that is vital and will make the article special.
> 
> But yes I believe I can.
> 
> I hope all this meets you well. Again sorry for late reply.

Lets just go ahead and close this one and submit a new topic form with the new idea. 👍 No rush - just want to clear up the backlog. 🚀

**Comment by Kikiodazie** [comment]
> > > Great! @Kikiodazie
> > > Can you add a unique idea to your article that makes it different from other blogs?
> > 
> > 
> > Hey @ahmadmardeni1 again so sorry for late reply.
> > Yes I believe so. Currently don't have that unique idea right now would have to go through other blogs and content and see what’s missing that is vital and will make the article special.
> > But yes I believe I can.
> > I hope all this meets you well. Again sorry for late reply.
> 
> Lets just go ahead and close this one and submit a new topic form with the new idea. 👍 No rush - just want to clear up the backlog. 🚀

No problem  @hectorkambow. Thanks.
